api/Dockerfile

```diff
@@ -17,4 +22,4 @@ EXPOSE 8000
 
 USER nobody
 
-CMD ["./bin/docker"]
+CMD ["./docker/docker"]
```

Comment:
Should the names change here as well perhaps? Seems a bit weird. Maybe `./scripts/run-docker` or something?

Reply:
Seems like an overcomplication to me?